code/datums/wounds/bones.dm

```diff
@@ -199,7 +199,17 @@
 	status_effect_type = /datum/status_effect/wound/blunt/moderate
 	scar_keyword = "bluntmoderate"
 
-/datum/wound/blunt/moderate/crush()
+/datum/wound/blunt/moderate/wound_injury(datum/wound/old_wound)
+	. = ..()
+	RegisterSignal(victim, COMSIG_LIVING_DOORCRUSH, .proc/door_crush)
+
+/datum/wound/blunt/moderate/Destroy()
+	if(victim)
+		UnregisterSignal(victim, COMSIG_LIVING_DOORCRUSH)
+	return ..()
```

Comment:
```suggestion

/datum/wound/blunt/moderate/Destroy()
	if(victim)
		UnregisterSignal(victim, COMSIG_LIVING_DOORCRUSH)
	return ..()


/datum/wound/blunt/moderate/wound_injury(datum/wound/old_wound)
	. = ..()
	RegisterSignal(victim, COMSIG_LIVING_DOORCRUSH, .proc/door_crush)

```
Try to leave the `New()`, `Initialize()` and `Destroy()` and related definitions at the top, as they set up the object's basic definition.